src/resultspage.cpp

```diff
@@ -212,6 +230,24 @@ void ResultsPage::onJumpToCallerCallee(const Data::Symbol& symbol)
     ui->resultsTabWidget->setCurrentWidget(m_resultsCallerCalleePage);
 }
 
+void ResultsPage::setData(const Data::Symbol& data)
+{
+    m_resultsDisassemblyPage->setSymbol(data);
+}
+
+void ResultsPage::setData(const Data::DisassemblyResult& data)
+{
+    m_resultsDisassemblyPage->setData(data);
+}
+
+void ResultsPage::onJumpToDisassembly(const Data::Symbol& symbol)
+{
+    ui->resultsTabWidget->addTab(m_resultsDisassemblyPage, tr("Disassembly"));
```

Comment:
what happens if this is called twice, for different symbols? ideally we should be able to show both to compare them

but for now just one would be fine, just make sure that we don't end up with two that show the same data?

Reply:
this is still not answered?

Reply:
Evething is OK for this. Each of them will show its own Disassemby data.

Reply:
how is that possible?  there is only one m_resultsDisassemblyPage, and it gets added multiple times? So I would assume that there will be multiple tabs all showing the same data?

for this to work as you explain, m_resultsDisassemblyPage would need to become a vector, and new pages added here?

Reply:
this has not been answered